### ISSUE TITLE
Type mismatch

### ISSUE THREAD

**Issue by siawyoung** [kxbmap/sbt-jooq#27]
Hi, I tried to set the configuration, but it gave me this error:

```
  lazy val jooqGen: Seq[Def.Setting[_]] = Seq(
    libraryDependencies ++= Seq(
      "io.circe" %% "circe-core" % Versions.circe,
      "javax.validation" % "validation-api" % Versions.validation,
      "org.postgresql" % "postgresql" % Versions.postgres % "jooq"
    ),
    jooqVersion := Versions.jooq,
    jooqCodegenConfig :=
      <configuration>
       .....
```

```
[error] /Users/tempuser/code/schema/project/build.scala:118:8: type mismatch;
[error]  found   : scala.xml.Elem
[error]  required: sbtjooq.codegen.CodegenConfig
[error]       <configuration>
[error]   
```

Any ideas? I was upgrading from 0.3.3 to 0.5.1 when this happened.

**Comment by kxbmap** [comment]
If you are writing build definitions in `.scala` file, you need to import implicit conversions.

```scala
import sbtjooq.codegen.JooqCodegenPlugin.autoImport._
```

Or you can write conversions explicitly.

```scala
jooqCodegenConfig := CodegenConfig.XML(<configuration>...</configuration>)
```

**Comment by kxbmap** [comment]
Released version 0.7.0 is enabled implicit conversions by default.
